src/features/user/TreeMapper/components/PlantLocationPage.tsx

```diff
@@ -259,37 +259,50 @@ export function LocationDetails({
             </div>
           </div>
         )}
-        {location.type === 'multi' && location.captureMode === 'on-site' && (
-          <div className={styles.singleDetail}>
-            <p className={styles.title}>{t('maps:sampleTree')}</p>
-            {/* <div className={styles.value}> */}
-            {location.samplePlantLocations &&
-              location.samplePlantLocations.map((spl: any, index: number) => {
-                return (
-                  <div key={index} className={styles.value}>
-                    {index + 1}.{' '}
-                    <span
-                      onClick={() => setselectedLocation(spl)}
-                      className={styles.link}
-                    >
-                      {spl.scientificName
-                        ? spl.scientificName
-                        : spl.scientificSpecies &&
-                          spl.scientificSpecies !== 'Unknown'
-                        ? spl.scientificSpecies
-                        : t('maps:unknown')}
-                    </span>
-                    <br />
-                    {spl.tag ? `${t('maps:tag')} #${spl.tag} • ` : null}
-                    {spl?.measurements?.height}
-                    {t('maps:meterHigh')} • {spl?.measurements?.width}
-                    {t('maps:cmWide')}
-                  </div>
-                );
-              })}
-            {/* </div> */}
-          </div>
-        )}
+        {location.type === 'multi' &&
+          location.captureMode === 'on-site' &&
+          location.samplePlantLocations.length > 0 && (
+            <div className={styles.singleDetail}>
+              <p className={styles.title}>{t('maps:sampleTree')}</p>
+              {/* <div className={styles.value}> */}
+              {location.samplePlantLocations &&
+                location.samplePlantLocations.map((spl: any, index: number) => {
+                  return (
+                    <div key={index} className={styles.value}>
+                      {index + 1}.{' '}
+                      <span
+                        onClick={() => setselectedLocation(spl)}
+                        className={styles.link}
+                      >
+                        {spl.scientificName
+                          ? spl.scientificName
+                          : spl.scientificSpecies &&
+                            spl.scientificSpecies !== 'Unknown'
+                          ? spl.scientificSpecies
+                          : t('maps:unknown')}
+                      </span>
+                      <br />
+                      {spl.status === 'dead' ? (
+                        <p>
+                          {spl?.status} • {spl?.statusReason}
+                        </p>
+                      ) : (
+                        ''
+                      )}
+                      {[...spl.history].reverse().map((h, index) => (
```

Comment:
Not sure if the history array will be sorted by created date.
Just to be on the safe side please use `moment` or `date-fns` library (which is already present in the project) to sort it by created date

Reply:
Yes history array is supposed to be sorted according to design provided.